app/app.iml

```diff
@@ -22,14 +22,13 @@
       </configuration>
     </facet>
     <facet type="kotlin-language" name="Kotlin">
-      <configuration version="3" platform="JVM 1.8" useProjectSettings="false">
```

Comment:
This file must NOT be checked into Version Control Systems, as it contains information specific to your local configuration. :)

Update your .gitignore file

Reply:
Done